CMakeLists.txt

```diff
@@ -13,25 +13,17 @@
 # limitations under the License.
 # ==============================================================================
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
You may want to lower the minimum version of cmake, default version of cmake in Ubuntu 18.04 is 3.10.2